content/application/summit.html

```diff
@@ -148,3 +151,34 @@ <h2 class="headline4 gmb-4">Agenda & further details coming soon!</h2>
   </div>
 </section>
 
+<section class="glue-page">
+  <div class="glue-grid gmb-4">    
+    <div class="col-span-4 sm:col-span-8 sm:col-start-3">
+      <h2 class="headline4 gpt-4 gmb-2">Event FAQ</h2>      
+      <h3 class="headline5 gpt-2 gmb-2">Getting There</h3>
+      <p class="gmb-2">
+        You can easily get to the venue via tube, bike or foot. It is a ~2 minute walk from <a href="https://maps.app.goo.gl/aRSybpgFsRSQ7CPG6">Kings Cross</a> and <a href="https://maps.app.goo.gl/MpPsnuoDUyEK4Yy9A">St Pancras</a> stations. 
+      </p>
+      <h3 class="headline5 gpt-2 gmb-2">Airport</h3>
+      <p class="gmb-2">
+        Airport
+If you haven’t visited London in a while (or before!), we have exciting news, the Elizabeth line is open! This gives a couple of choices for getting to the venue either with the Elizabeth line or Piccadilly: <a href="https://maps.app.goo.gl/yvJrzmcqGe6ZqfuD6">Heathrow to Google 6PS</a> (~60 minutes).
+      </p>
+      <h3 class="headline5 gpt-2 gmb-2">Room Block</h3>      
+      <p class="gmb-1">
+        We do not have a room block with discounted rooms, but there are plenty of great places to stay nearby, including:
```

Comment:
I've also heard good reviews of this one: https://maps.app.goo.gl/UdVqc54jujpQkZJC6, it's a good compromise between the two others: much cheaper than the Great Northern, a bit more expensive than EasyHotel, but much closer.

Reply:
Great, it will be good to have a 3rd option - will add it in.